PythonPonies/bangazon-storefront
# Ike product types model

# Description
This PR adds the product types model and tests the model is persisting data

## Number of Fixes
1

## Related Ticket(s)
products_model,

## Problem to Solve
Describe the problem(s) this pull request solves.
Persisting product type data

## Proposed Changes
Describe the proposed changes.
Add product_types_model and test that the model works properly
## Expected Behavior
Describe the expected behavior.
Add products_type to the datatable

## Steps to Test Solution
Drop the database / delete database file

Within the root:
Run `python manage.py makemigrations`
run `python manage.py migrate`

a.
`git fetch --prune`
switch into branch named `ike-productTypes-setup`
ensure test productTypes_model passes
1. Check the migrations file and ensure the productTypes table is created:

## Testing

[x ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ x] I certify that all existing tests pass

## Documentation
[ x] There is new documentation in this pull request that must be reviewed..
[ x] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
